new/LIT7038Qene.xml

```diff
@@ -0,0 +1,82 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7038Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">እስመ፡ ወርኃ፡ ተድላ፡ ኮኖሙ፡ ለአዕባነ፡ ፍኖት፡ አፈ፡</title>
```

Comment:
It seems like the second line is now used as the title instead of the first, or am I mistaken?

Reply:
I took the first line of which carries the rhyme, but I can also change or use both.